.github/workflows/ci.yml

```diff
@@ -87,7 +87,12 @@ jobs:
           # requires two github secrets in repo to activate; see ci/github/coverity.sh
           # does not run on pull requests, only on pushes into develop and master
           - { name: Coverity, coverity: yes,
-              compiler: clang-10,  cxxstd: '17',             os: ubuntu-20.04 }
+              compiler: clang-10,  cxxstd: '17',             os: ubuntu-20.04, ccache: no }
```

Comment:
I'd put the `, ccache: no` into the first line next to the feature that requires this, i.e. `name: Coverity, coverity: yes, ccache: no,`. Might be easer to understand or what do you say?